ui/src/components/AccordionTimeline/AccordionTimeline.tsx

```diff
@@ -0,0 +1,87 @@
+import * as AccordionPrimitive from '@radix-ui/react-accordion';
+
+import Button from '@/components/Button/Button';
+import { IconChevron } from '@/icons';
+import classNames from '@/utils/classnames';
+
+type AccordionTimelineProps = {
+  timelineItems: AccordionTimelineItemProps[];
+};
+
+export default function AccordionTimeline({ timelineItems }: AccordionTimelineProps) {
+  return (
+    <AccordionPrimitive.Root
+      type="multiple"
+      className="text-slate-100 w-full last:border-b last:border-slate-800/50"
+    >
+      {timelineItems &&
+        timelineItems.map((item) => {
+          if (!item) return <p></p>;
+          const { id, header, expandable, position, content } = item;
+          return (
+            <AccordionTimelineItem
+              id={id}
```

Comment:
The `key` prop needs to be moved to where we consume `AccordionTimelineItem` within a `map`, rather than within `AccordionTimelineItem`

```suggestion
              id={id}
              key={id}
```

Reply:
Oh, good catch! I extracted the component but didn't change that!